### DISCUSSION TITLE
Developer parameter for bolt spacing in Splice operation not working, anyone can help on this?

### DISCUSSION THREAD

**Discussion by Jaimin6196** [idea-statica/ideastatica-public#252] [Help Q&A] [answered]
![image](https://user-images.githubusercontent.com/91888611/224988243-f825526b-e2e0-41e9-afbf-399053ad36e8.png)

**Comment by nathanlukeIDEA** [comment]
@Jaimin6196 Bolt spacing model properties are currently not a functioning part of what is possible to manipulate with parameters, although we are currently in progress of cleaning and improving this functionality. 

I have created an issue for this.

**Reply by Jaimin6196** [reply]
Okay great, thanks for the prompt reply @nathanlukeIDEA . Also, I've one more request, a tapered beam addition would be much helpful as we encounter it way often, for now using the way around with T-section and Widener.

Thanks, appreciate your help.
Cheers,
Jaimin

**Reply by nathanlukeIDEA** [reply]
@Jaimin6196  We understand this and it is something we are looking into. We can handle tapers in IOM they are just not currently imported into the IDEA connection.

**Reply by Jaimin6196** [reply]
I see, Thanks for the update @nathanlukeIDEA . Good to know it's available in IOM.

**Comment by nathanlukeIDEA** [comment, marked as answer]
@Jaimin6196  In version 23 of IDEA StatiCa, the parameters interface has been improved significantly. You should now be able to define all anchor geometry relatively easily. Changing bolt sizes is also possible.

https://www.ideastatica.com/support-center/setting-up-parametric-design-with-developer-tab
